Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -544,6 +544,18 @@ TEST_CASE(test_tiff_ccitt_rle)
     EXPECT_EQ(frame.image->get_pixel(60, 75), Gfx::Color::NamedColor::Black);
 }
 
+TEST_CASE(test_tiff_ccitt3)
+{
+    auto file = TRY_OR_FAIL(Core::MappedFile::map(TEST_INPUT("tiff/ccitt3.tiff"sv)));
+    EXPECT(Gfx::TIFFImageDecoderPlugin::sniff(file->bytes()));
+    auto plugin_decoder = TRY_OR_FAIL(Gfx::TIFFImageDecoderPlugin::create(file->bytes()));
```

Comment:
Can you say how you created the test file, or where you got it from, in the commit message?